test/integration/raiden.test.ts

```diff
@@ -4,7 +4,7 @@ import * as path from "path";
 import * as fse from "fs-extra";
 import net from "net";
 
-import waitPort from "wait-port";
+const waitPort: any = require("wait-port");
```

Comment:
wait-port does not have proper type declarations; using _require_ allows to disable the type checking.